src/site/content/en/blog/2021-scroll-survey/index.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: 2021 Scroll Survey
```

Comment:
(Sorry, I can't remember if I said this already)

This should be a call-to-action
```suggestion
title: Take the 2021 scroll survey to help improve scrolling on the web
```